otter/models/intents.py

```diff
@@ -14,3 +19,35 @@ class ModifyGroupState(object):
 def perform_modify_group_state(dispatcher, mgs_intent):
     """Perform a :obj:`ModifyGroupState`."""
     return mgs_intent.scaling_group.modify_state(mgs_intent.modifier)
+
+
+@attributes(['tenant_id', 'group_id'])
+class GetScalingGroupInfo(object):
+    """Look up a scaling group, along with its `state` and launch config."""
+
+
+@deferred_performer
+@inlineCallbacks
+def perform_get_scaling_group_info(log, store, dispatcher, intent):
+    """
+    Perform :obj:`GetScalingGroupInfo`.
+
+    :param log: bound log
+    :param IScalingGroupCollection store: collection of groups
+    :param dispatcher: dispatcher provided by perform
+    :param GetScalingGroupInfo intent: the intent
+    """
+    group = yield store.get_scaling_group(
+        log, intent.tenant_id, intent.group_id)
+    state = yield group.view_state()
+    lc = yield group.view_launch_config()
+    returnValue((group, state, lc))
```

Comment:
Yes, in the code that actually uses this intent, we then go on to use ModifyGroupState.